src/types/onyx/LastPaymentMethod.ts

```diff
@@ -1,4 +1,18 @@
+/**
+ * The new lastPaymentMethod object
+ */
+type LastPaymentMethodType = {
+    /** The default last payment method, this one holds the existing data of the old lastPaymentMethod value which is string */
```

Comment:
I think this is confusing, we didn't migrate to the new structure right?

```suggestion
    /** The default last payment method */
```